<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkspaceWizard</class>
 <widget class="QWidget" name="WorkspaceWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Welcome to the Workspace Wizard</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ForwardButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BackButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Directions">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>361</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Press &quot;Next&quot;' to continue</string>
   </property>
  </widget>
  <widget class="QPushButton" name="QuestionButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>371</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="ItemLayout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
